aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -1227,7 +1227,7 @@ std::pair<Tensor, hidden_type> _cudnn_impl(
   auto weight_buf = try_get_weight_buf(
       input, params, has_biases, mode, hidden_size, num_layers, bidirectional);
   if (!weight_buf.defined()) {
-    AT_WARN(WEIGHT_FORMAT_WARN);
+    // AT_WARN(WEIGHT_FORMAT_WARN);
```

Comment:
and here. If there's a good reason for commenting it out, let's add a TODO with an explanation of why and when the line can be re-enabled. If it's meant to never be re-enabled, we should remove the line.

Reply:
ack